src/ssmDocument/commands/openDocumentItem.ts

```diff
@@ -38,7 +38,9 @@ export async function openDocumentItem(node: DocumentItemNode, awsContext: AwsCo
             content: rawContent.Content,
             language: `ssm-${rawContent.DocumentFormat!.toLowerCase()}`,
         })
-        vscode.window.showTextDocument(textDocument)
+        if (textDocument !== undefined) {
```

Comment:
I think this if statement renders the catch block mostly useless? Which means we won't get a failure event.

Reply:
Should be changed to `await` instead of an if statement.